tests/back/api/test_global_explanation_api.py

```diff
@@ -0,0 +1,204 @@
+                "url": "",
+            },
+            files={"file": ("filename", csv_file, "text/csv")},
+        )
+    assert response.status_code == 201, response.text
+    dataset = response.json()
+    yield dataset["id"]
+    response = client.delete(f"/api/v1/dataset/{dataset['id']}")
+    assert response.status_code == 204, response.text
+
+
+@pytest.fixture(scope="module", name="experiment_id")
+def fixture_experiment_id(session: sessionmaker, dataset_id: int):
+    db = session()
+    dummy_experiment = Experiment(
+        dataset_id=dataset_id,
+        task_name="TabularClassificationTask",
+        name="Test Experiment",
+    )
+    db.add(dummy_experiment)
+    db.commit()
+    db.refresh(dummy_experiment)
+    yield dummy_experiment.id
+
+    db.delete(dummy_experiment)
+    db.commit()
+
+
+@pytest.fixture(scope="module", name="run_id_1")
+def fixture_run_id_1(session: sessionmaker, experiment_id: int):
+    db = session()
+    run = Run(
+        experiment_id=experiment_id,
+        model_name="RandomForestClassifier",
+        parameters={},
+        name="Run",
+    )
+    db.add(run)
+    db.commit()
+    db.refresh(run)
+
+    yield run.id
+
+    db.delete(run)
+    db.commit()
+    db.close()
+
+
+@pytest.fixture(scope="module", name="run_id_2")
+def fixture_run_id_2(session: sessionmaker, experiment_id: int):
+    db = session()
+    run = Run(
+        experiment_id=experiment_id,
+        model_name="SVC",
+        parameters={},
+        name="Run",
+    )
+    db.add(run)
+    db.commit()
+    db.refresh(run)
+
+    yield run.id
+
+    db.delete(run)
+    db.commit()
+    db.close()
+
+
+def test_create_global_explanation(client: TestClient, run_id_1: int, run_id_2):
+    response = client.post(
+        "/api/v1/global-explanation/",
+        json={
+            "name": "test_1",
+            "run_id": run_id_1,
+            "explainer_name": "PartialDependence",
+            "parameters": {"grid_resolution": 100, "percentiles": [0.1, 0.6]},
+        },
+    )
+    assert response.status_code == 201, response.text
+
+    response = client.post(
+        "/api/v1/global-explanation/",
+        json={
+            "name": "test_2",
+            "run_id": run_id_1,
+            "explainer_name": "PartialDependence",
+            "parameters": {"grid_resolution": 50, "percentiles": [0.1, 0.2]},
+        },
+    )
+    assert response.status_code == 201, response.text
+
+    response = client.post(
+        "/api/v1/global-explanation/",
+        json={
+            "name": "test_3",
+            "run_id": run_id_2,
+            "explainer_name": "PartialDependence",
+            "parameters": {"grid_resolution": 50, "percentiles": [0.1, 0.55]},
+        },
+    )
+    assert response.status_code == 201, response.text
+
+
+def test_get_global_explanation_by_id(
+    client: TestClient, run_id_1: int, run_id_2: int, dataset_id: int
+):
+    response = client.get("/api/v1/global-explanation/1")
```

Comment:
Now the tests are independent of each other (or should be), so here, before testing the get, you should make a request to create the explainer.